.openpublishing.redirection.json

```diff
@@ -242,6 +242,26 @@
     {
         "source_path": "docs/tutorials/index.md", 
         "redirect_url": "/dotnet/articles/samples-and-tutorials/"
+    },
+    {
+        "source_path": "docs/csharp/getting-started/consuming-library-with-visual-studio-2017.md", 
+        "redirect_url": "/dotnet/articles/getting-started/consuming-library-with-visual-studio",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs/csharp/getting-started/debugging-with-visual-studio-2017.md", 
```

Comment:
I couldn't rename that one (or the publishing one). The `debugging-with-visual-studio.md` and the `publishing-with-visual-studio.md` are 301 redirecting.

Did you want to ignore that and move the content over to those files anyway?